API_Quick_Start.md

```diff
@@ -90,22 +90,24 @@ http https://data-hub.eglobalmark.com/ngsi-ld/v1/entities type==BeeHive \
 * Let's add a name to the created beehive: 
 
 ```
-http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01/attrs Content-Type:application/json \
+http POST https://data-hub.eglobalmark.com/ngsi-ld/v1/entities/urn:ngsi-ld:BeeHive:01/attrs \
     Link:"<https://raw.githubusercontent.com/easy-global-market/ngsild-api-data-models/master/apic/jsonld-contexts/apic-compound.jsonld>; rel=http://www.w3.org/ns/json-ld#context; type=application/ld+json" \
+    Content-Type:application/json \

```

Comment:
application/json is the default content type (no need to specify it): https://httpie.org/docs#default-behaviour